ee/tabby-db/migrations/0025_user-events.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE user_events (
+    id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    user_id INTEGER NOT NULL,
+    completion_id TEXT NOT NULL,
+    type TEXT NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT (DATETIME('now')),
+    payload TEXT NOT NULL
```

Comment:
json field

Reply:
SQLite has no JSON type

Reply:
https://www.sqlite.org/changes.html#version_3_45_0

jsonb column has landed since 3.45

Reply:
![image](https://github.com/TabbyML/tabby/assets/14198267/91b5bf32-f760-4964-aa28-ff26bcfbb103)
Using sqlite version 3.45.3

Reply:
the data column for it should just be blob?